pkg/cmd/commands.go

```diff
@@ -56,6 +56,12 @@ func CreateArtifacts(paths []string, generate bool, args ...string) error {
 		}
 
 		ros, includeResources, err := spec.CoreOperations(kedgeData)
+
+		// Validate input kedge file
+		if !skipValidation {
```

Comment:
shouldn't we move this before `CoreOperations`?

Reply:
Yeah , that will save much time and operation